<?xml version="1.0" ?>

<launch>

    <node pkg = "roboteq_controller" type = "roboteq_controller_node" name = "roboteq_controller" ns="roboteq" output = "screen">
        <rosparam command="load" file="$(find roboteq_controller)/config/query.yaml" />
    </node>

    <node pkg="roboteq_controller" type="roboteq_proc_node.py" name="roboteq_proc" ns="roboteq" output="screen" />

</launch>
